shinylive/_utils.py

```diff
@@ -58,12 +58,16 @@ def listdir_recursive(dir: str | Path) -> list[str]:
     return all_files
 
 
+FromTo = Tuple[str, str]
+
+
 def copy_file_and_substitute(
-    src: str | Path, dest: str | Path, search_str: str, replace_str: str
+    src: str | Path, dest: str | Path, *, replacements: tuple[FromTo, ...]
```

Comment:
A `tuple` here seems too restrictive, it's plausible that someone would want to pass in a list.

```suggestion
    src: str | Path, dest: str | Path, *, replacements: Sequence[FromTo, ...]
```